lib/csp.js

```diff
@@ -72,6 +72,7 @@ function addInlineScriptExceptions (directives) {
   // TODO: This is the SHA-256 hash of the inline script in build/reveal.js/plugins/notes/notes.html
   // Any more clean solution appreciated.
   directives.scriptSrc.push('\'sha256-EtvSSxRwce5cLeFBZbvZvDrTiRoyoXbWWwvEVciM5Ag=\'')
+  directives.scriptSrc.push('\'sha256-L0TsyAQLAc0koby5DCbFAwFfRs9ZxesA+4xg0QDSrdI=\'')
```

Comment:
Shouldn't this one replace the upper one?

Reply:
Yeah, maybe. Sorry I couldn't remember it.
And seems it affect by when upgrade the reveal.js if it comes with changes in `notes.html`